packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -2,30 +2,89 @@
     <div
         :class="$style['c-rating']"
         data-test-id="rating-component">
-        {{ copy.text }}
+        <ul :class="$style['c-rating-star-wrapper']">
+            <li
+                v-for="star in maxStarRating"
+                :key="star"
+                :class="$style['c-rating-star']">
+                <star-filled-icon
+                    v-if="getRating(star)"
+                    :class="$style['c-rating-star-filled']" />
+                <star-icon
+                    v-else
+                    :class="$style['c-rating-star-empty']" />
+            </li>
+            <span
```

Comment:
I don't think this will be valid HTML – I would think the `span` will need to sit outside the `ul` tag

Reply:
Fixed